### DISCUSSION TITLE
发送邮件

### DISCUSSION THREAD

**Discussion by koi-pig** [dreamhunter2333/cloudflare_temp_email#522] [General]
这个算记录已经添加了吧,但是那个resend里面还是显示添加失败
![image](https://github.com/user-attachments/assets/188626ff-2f8d-43a4-a8e7-199fd5624931)
![image](https://github.com/user-attachments/assets/0dae3769-2975-45e5-a377-5171473dc5cb)
![image](https://github.com/user-attachments/assets/7597ba09-02df-4791-ac95-0f13c9e68985)

**Comment by 3377** [comment]
看了一遍action部署，只能说代码写的好的，文档写的是真的烂

**Comment by dreamhunter2333** [comment]
不确定 resend 是否支持二级域名，正常的域名是没问题的

**Comment by dreamhunter2333** [comment]
参考 #515
